docs/manuals/core/versioning/development_mode.md

```diff
@@ -0,0 +1,27 @@
+When developing a Taipy Core application, data nodes, tasks, pipelines, ... or the execution flow are constantly changing.
```

Comment:
I don't get the `...`, is it missing some text?

Reply:
```suggestion
When developing a Taipy Core application, data nodes, tasks, pipelines, etc. or the execution flow are constantly changing.
```
It's like 'etc'.